### DISCUSSION TITLE
igl::spectra::lscm produces overlapping UVs

### DISCUSSION THREAD

**Discussion by wtholliday** [libigl/libigl#2329] [Q&A] [answered]
On a few of my charts, `igl::spectra::lscm` produces overlapping UVs:

<img width="817" alt="IMG_6567" src="https://github.com/libigl/libigl/assets/841144/5caa099c-8bfb-4230-9c4c-9517ea23042f">

Wondering how to deal with that? I can post the mesh if necessary.

**Comment by alecjacobson** [comment, marked as answer]
there's no guarantee that this won't happen with LSCM.

igl::slim* https://libigl.github.io/dox/namespaceigl.html#a4b29382a965c510055a154aac6ecde4e is a method that can prevent flips/overlaps
